tests/testthat/test-applications.R

```diff
@@ -20,6 +20,9 @@ test_succeeds("keras pre-built models can be instantiated", {
   application_vgg16()
   application_vgg19()
   application_inception_v3()
+  
+  application_densenet()
```

Comment:
You may need a version conditional for these applications as well (e.g. https://github.com/rstudio/keras/blob/master/tests/testthat/test-applications.R#L16-L18)